### DISCUSSION TITLE
Does WingetUI come bundled with Chocolatey?

### DISCUSSION THREAD

**Discussion by andresestrella** [Devolutions/UniGetUI#1339] [Q&A] [answered]
When looking through my installed packages I see I have Chocolatey 1.4.0 installed, but I can't remember ever having installed it in my machine (and can't seem to find where the install folder is on my drive). 
WingetUI also allows me to install Chocolatey packages, I'm assuming using that chocolatey 1.4 executable.

**Comment by marticliment** [comment, marked as answer]
Yes, chocolatey comes bundled with WingetUI. However, if you happened to have Chocolatey already installed, WingetUI would have detected that and used System Chocolatey by default

**Reply by andresestrella** [reply]
do you know where this bundled version gets installed to? I would like to set a PATH env var to it so I could use it on the command line as well

**Reply by marticliment** [reply]
C:\Users\user\AppData\Local\Programs\WingetUI\choco-cli\bin


note: on the next version, chocolatey will be automatically added to path by WingetUI

**Reply by andresestrella** [reply]
great! thanks
